packages/i18n/src/resources/zh-Hans.json

```diff
@@ -133,6 +134,7 @@
   "Share with link": "通过链接分享",
   "Copy Link": "复制链接",
   "Publishing Description": "发布到web后，所有人都可以通过链接查看此工作区的内容。",
+  "Export Description": "您可以导出整个Workspace数据进行备份，并且可以重新导入导出的数据。",
```

Comment:
The Chinese translation does not need to be added, it is downloaded from the translation platform.